cuqi/geometry/_geometry.py

```diff
@@ -702,32 +702,80 @@ def _compute_eigpairs(self, xnod, C_nu, sigma, M, N_GL):
 
 class StepExpansion(Continuous1D):
     '''
-    class representation of the step random field with 3 intermidiate steps
-    '''
-    def __init__(self, grid, axis_labels=['x'], **kwargs):
+    Class representation of step functions (piecewise constant functions) with `n_steps` 
+    equidistant steps on the interval [x0, xn], of length L.
```

Comment:
```suggestion
    equidistant steps on the interval [x0, xn], with both endpoints included.
```

Reply:
I think it is no longer relevant to the user whether we refer to the interval length as `L` now that it is automatically computed from x0 and xn?